### ISSUE TITLE
mind/var/special_role is not set properly on autotraitor

### ISSUE THREAD

**Issue by OneOneThreeEight** [Aurorastation/Aurora.3#578]
Definitely confirmed on AutoTraitor, but it will identify team-specific antagonists such as SyndiCakes, Heistbirds, Cultists, ERT/loyalists, and so on. IIRC, AntagHUD has had issues identifying vampires and changelings for the past couple weeks as well.

**Comment by skull132** [comment]
Fugging name modifications.

Proper issue shown.

**Comment by OneOneThreeEight** [comment]
@skull132 Had a really weird testing period with this today.

Changed a dummy's antag status a few times, toggled things back off and then back on, had my own antag HUD enabled for posterity. I was able to see vampires, vampire thralls, changelings and traitor, if I directly modified the mob's special role status. With their respective HUD icons, ofc, in the top right of the mob sprite.

I am unsure why this is the case for autotraitor, however. I'm probably thinking it has something specifically to do with how the game mode handles assignment of the special roles.
